modules/virt-creating-bridge-nad-web.adoc

```diff
@@ -8,7 +8,7 @@
 [id="virt-creating-bridge-nad-web_{context}"]
 = Creating a Linux bridge network attachment definition in the web console
```

Comment:
NetworkAttachmentDefinition ?

Reply:
According to our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting), if you are referring to the actual instance of the API object, use NetworkAttachmentDefinition . If it is a general or conceptual reference, use "network attachment definition".

In this case, it is a general reference, so we use "network attachment definition".

Reply:
Pan's explanation settled this one for me.